platform/linuxbsd/export/export_plugin.cpp

```diff
@@ -67,7 +67,7 @@ Error EditorExportPlatformLinuxBSD::export_project(const Ref<EditorExportPreset>
 	if (!template_path.is_empty()) {
 		String exe_arch = _get_exe_arch(template_path);
 		if (arch != exe_arch) {
-			add_message(EXPORT_MESSAGE_ERROR, TTR("Prepare Templates"), vformat(TTR("Mismatching custom export template executable architecture, found \"%s\", expected \"%s\"."), exe_arch, arch));
+			add_message(EXPORT_MESSAGE_ERROR, TTR("Prepare Templates"), vformat(TTR("Mismatching custom export template executable architecture: found \"%s\", expected \"%s\"."), exe_arch, arch));
```

Comment:
```suggestion
			add_message(EXPORT_MESSAGE_ERROR, TTR("Prepare Templates"), vformat(TTR(R"(Mismatching custom export template executable architecture: found "%s", expected "%s".)"), exe_arch, arch));
```
While we're here we might improve this with raw strings

Reply:
The translation extraction script can't handle raw string literals currently.

Reply:
This should probably be documented somewhere